packages/web/src/components/userprofile/UserProfile.tsx

```diff
@@ -12,15 +14,21 @@ interface User {
 export interface UserProfileProps {
   user: User;
   setUser(user: User): void;
+  isCurrentUser: boolean;
 }
 
-export const UserProfile: React.FC<UserProfileProps> = ({ user, setUser }: UserProfileProps) => {
+export const UserProfile: React.FC<UserProfileProps> = ({
+  user,
+  setUser,
+  isCurrentUser,
+}: UserProfileProps) => {
+  const [editToggle, setEditToggle] = React.useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = React.useState<string>('');
```

Comment:
Same thing for these
```suggestion
  const [editToggle, setEditToggle] = React.useState(false);
  const [errorMessage, setErrorMessage] = React.useState('');
```